src/constants.js

```diff
@@ -1,75 +1,65 @@
 import * as styledSystem from 'styled-system'
+import { compose } from 'styled-system'
+import systemPropTypes from '@styled-system/prop-types'
+import themeGet from '@styled-system/theme-get'
 import theme from './theme'
 
-const {compose, get: getKey, themeGet} = styledSystem
+const { get: getKey } = styledSystem
 
 export const get = key => themeGet(key, getKey(theme, key))
 
-export const COMMON_LIST = ['color', 'space']
-
-export const BORDER_LIST = ['borders', 'borderColor', 'boxShadow', 'borderRadius']
-
-export const TYPOGRAPHY_LIST = [
-  // typography props
-  'fontFamily',
-  'fontSize',
-  'fontStyle',
-  'fontWeight',
-  'lineHeight',
-  'textAlign'
-]
+export const COMMON = compose(
+  styledSystem.space,
+  styledSystem.color,
```

Comment:
I think we recently added `display` to this list.

Reply:
yes we did but it hasn't merged yet. I'll make that change here too so it merges into the release branch fine